### PR TITLE
Updates FreqDist docs with plot directive and datasets module

### PR DESCRIPTION
**Pull request by rebeccabilbro**
This PR addresses portions of #687 and #683, which requested that the documentation be updated to use the new plot directive and to use the new datasets module in our docs.

I have made the following changes    
1. Updated `freqdist.rst` to use the plot directive and hobbies corpus from the new datasets module. _Note that I elected to suppress the code blocks for the last three of the four images that are generated, since it is very repetitive. I have also used a mix of `orient='h'` and `orient='v'` for variety. I also had to specify the image size for the two horizontally oriented plots since the plot directive was squishing the labels a bit._
2. Removed `freqdist.py` since it is no longer need to generate the images.
3. Removed `freqdist_cooking.png`, `freqdist_corpus.png`, `freqdist_gaming.png`, and `freqdist_stopwords.png` since these images will be generated via the plot directive when the docs are built.

### CODE REVIEW

File: docs/api/text/freqdist.rst [hunk 1]
```diff
@@ -5,74 +5,119 @@ Token Frequency Distribution
 
 A method for visualizing the frequency of tokens within and across corpora is frequency distribution. A frequency distribution tells us the frequency of each vocabulary item in the text. In general, it could count any kind of observable event. It is a distribution because it tells us how the total number of word tokens in the text are distributed across the vocabulary items.
 
-.. code:: python
+Note that the ``FreqDistVisualizer`` does not perform any normalization or vectorization, and it expects text that has already be count vectorized.
```
Comment (lwgray) [inline docs/api/text/freqdist.rst]: This is only a suggestion but consider using a restructuredtext NOTE format here `.. note:: `

File: docs/api/text/freqdist.rst [hunk 2]
```diff
@@ -5,74 +5,119 @@ Token Frequency Distribution
...
-.. code:: python
+    # Load the text data
+    corpus = load_hobbies()
 
     vectorizer = CountVectorizer()
     docs       = vectorizer.fit_transform(corpus.data)
     features   = vectorizer.get_feature_names()
 
-    visualizer = FreqDistVisualizer(features=features)
+    visualizer = FreqDistVisualizer(features=features, orient='v')
```
Comment (lwgray) [inline docs/api/text/freqdist.rst]: Great!  It is good to give the user some variety and also showcase different parameters.

File: docs/api/text/freqdist.rst [hunk 3]
```diff
@@ -5,74 +5,119 @@ Token Frequency Distribution
...
+    )
     visualizer.fit(docs)
     visualizer.poof()
 
-.. image:: images/freqdist_cooking.png
+And for the gaming corpus (again oriented horizontally):
+
+.. plot::
+    :context: close-figs
+    :include-source: False
```
Comment (lwgray) [inline docs/api/text/freqdist.rst]: Great use this option!

File: docs/api/text/freqdist.rst [hunk 4]
```diff
@@ -5,74 +5,119 @@ Token Frequency Distribution
...
+
+    from yellowbrick.text import FreqDistVisualizer
+    from yellowbrick.datasets import load_hobbies
 
-.. code:: python
+    # Load the text data
+    corpus = load_hobbies()
+
+    hobbies = defaultdict(list)
+    for text, label in zip(corpus.data, corpus.target):
```
Comment (lwgray) [inline docs/api/text/freqdist.rst]: Maybe add a comment here as to what you are doing with this line of code.

File: docs/api/text/freqdist.rst [hunk 5]
```diff
@@ -5,74 +5,119 @@ Token Frequency Distribution
...
+
+    from yellowbrick.text import FreqDistVisualizer
+    from yellowbrick.datasets import load_hobbies
+
+    # Load the text data
+    corpus = load_hobbies()
+
     hobbies = defaultdict(list)
-    for text, label in zip(corpus.data, corpus.label):
+    for text, label in zip(corpus.data, corpus.target):
```
Comment (lwgray) [inline docs/api/text/freqdist.rst]: Maybe add a comment here as to what you are doing with this line of code.
